packages/android_intent_plus/android/src/main/java/dev/fluttercommunity/plus/androidintent/IntentSender.java

```diff
@@ -98,6 +98,9 @@ boolean canResolveActivity(Intent intent) {
     }
 
     final PackageManager packageManager = applicationContext.getPackageManager();
+    boolean result =
+        packageManager.resolveActivity(intent, PackageManager.MATCH_DEFAULT_ONLY) != null;
```

Comment:
I don't understand the change, where are you using this `result`?

Reply:
I used it for debugging purposes. I'll remove it. Thanks for catching it.